CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+## Code of Conduct
+
+Please note that this project is released with a [Code of Conduct](./CODE_OF_CONDUCT.md).
+By participating in this project you agree to abide by its terms.
+
+## Matrix room
+
+Joining the matrix room at [#codespan:matrix.org][codespan-matrix] is a good way to get in touch with the developers and community.
+
+[codespan-matrix]: https://app.element.io/#/room/#codespan:matrix.org
+
```

Comment:
```suggestion

## Reporting bugs & suggesting improvements

Bugs (unwanted behaviour) and suggested improvements are tracked as GitHub issues.

Before reporting an issue, please check the following points:

1. The issue is **caused by `codespan-reporting` itself** and not by how it is used. Have a look at the documentation if you are not sure. If the documentation is not helpful, you can contact the developers at the above matrix chat address.
1. Your issue has **not already been reported** by someone else. Please look through the open issues in the issue tracker.

When reporting an issue, please add as much relevant information as possible. This will help developers and maintainers to resolve your issue. Some things you might consider:

* Use a descriptive title.
* Describe how a problem can be reproduced. Ideally give a minimal example.
* Explain what exactly is the problem and what you expect instead. If it is related to rendering, add screenshots or other illustrations.
```